resources/assets/components/blocks/ScholarshipInfoBlock/ScholarshipInfoBlock.js

```diff
@@ -56,12 +60,22 @@ const ScholarshipInfoBlock = ({
   numberOfScholarships,
   utmLabel,
 }) => {
-  const { loading, error, data } = useQuery(SCHOLARSHIP_AFFILIATE_QUERY, {
-    variables: {
-      utmLabel,
-      preview: env('CONTENTFUL_USE_PREVIEW_API'),
-      campaignId,
-    },
+  let loading;
+  let error;
+  let data;
+
+  if (utmLabel) {
+    ({ loading, error, data } = useQuery(SCHOLARSHIP_AFFILIATE_QUERY, {
+      variables: {
+        utmLabel,
+        preview: env('CONTENTFUL_USE_PREVIEW_API'),
+        campaignId,
+      },
+    }));
+  }
```

Comment:
This might read a little more cleanly using the [`skip` parameter](https://www.apollographql.com/docs/react/api/react-hooks/#params) for the `useQuery` hook:

```suggestion
  let loading;
  let error;
  let data;

const { loading, error, data } = useQuery(SCHOLARSHIP_AFFILIATE_QUERY, {
  skip: utmLabel === null,
  variables: {
    utmLabel,
    preview: env('CONTENTFUL_USE_PREVIEW_API'),
    campaignId,
  },
}));
```